src/commands/images/pushImage.ts

```diff
@@ -5,48 +5,69 @@
 
 import vscode = require('vscode');
 import { IActionContext } from 'vscode-azureextensionui';
-import { configurationKeys } from '../../constants';
 import { ext } from '../../extensionVariables';
 import { localize } from '../../localize';
 import { ImageTreeItem } from '../../tree/images/ImageTreeItem';
-import { askToSaveRegistryPath } from '../registries/registrySettings';
+import { registryExpectedContextValues } from '../../tree/registries/registryContextValues';
+import { RegistryTreeItemBase } from '../../tree/registries/RegistryTreeItemBase';
 import { addImageTaggingTelemetry, tagImage } from './tagImage';
 
 export async function pushImage(context: IActionContext, node: ImageTreeItem | undefined): Promise<void> {
     if (!node) {
         node = await ext.imagesTree.showTreeItemPicker<ImageTreeItem>(ImageTreeItem.contextValue, {
             ...context,
-            noItemFoundErrorMessage: localize('vscode-docker.commands.images.push.noImages', 'No images are available to push')
+            noItemFoundErrorMessage: localize('vscode-docker.commands.images.push.noImages', 'No images are available to push'),
```

Comment:
Not needed but I kinda prefer to keep it; if you want to add more stuff to this list then it won't necessitate a line change to this one. Same idea as leaving commas after the last item in an enum, so that adding one thing doesn't cause two line changes:
```javascript
enum Fruit {
    Apple,
    Banana,
    Orange,
}
```

Reply:
fair enough